app/models/settings/community.rb

```diff
@@ -7,7 +7,17 @@ class Community < Base
             default: ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"] || Time.zone.today.year
     setting :community_description, type: :string
     setting :community_emoji, type: :string, default: "🌱", validates: { emoji_only: true }
-    setting :community_name, type: :string, default: ApplicationConfig["COMMUNITY_NAME"] || "New Forem"
+    setting(
+      :community_name,
+      type: :string,
+      default: ApplicationConfig["COMMUNITY_NAME"] || "New Forem",
+      validates: {
+        format: {
+          with: /\A[^[<|>]]+\Z/,
+          message: "may not include the \"<\" nor \">\" character"
```

Comment:
@jeremyf Totally minor but this would have been a good use-case for a single-quoted string so you don't have to escape the double quotes. Rubocop is ok with that too.